spec/dummy/app/assets/stylesheets/preview.scss

```diff
@@ -17,6 +17,10 @@ body {
   color: #222;
 }
 
+.page_header {
```

Comment:
Yep, I think I was second-guessing myself and assuming that there was a reason it wasn't already there.